tests/features/communities/eif/vocabulary.feature

```diff
@@ -45,3 +45,34 @@ Feature:
     When I go to "/solutions"
     Then I should see the "Some EIF solution" tile
     But I should not see the text "EIF reference"
+
+  @terms
+  Scenario: Solutions referencing an EIF term should appear in the corresponding page.
+    Given collection:
+      | title | EIF Toolbox |
+      | state | validated   |
+    And solutions:
+      | title    | state     | policy domain | collection  | eif reference                                                                             |
+      | Balker   | validated |               | EIF Toolbox | Underlying Principle 1: subsidiarity and proportionality, Underlying Principle 2: openess |
+      | Corridor | validated |               | EIF Toolbox | Underlying Principle 1: subsidiarity and proportionality                                  |
+      | Lager    | validated | Demography    | EIF Toolbox |                                                                                           |
+
+    # Underlying Principle 1.
+    When I go to "/taxonomy/term/http_e_f_fdata_ceuropa_ceu_fUP1"
```

Comment:
When I go to this page I see the page title twice. We should hide the one in the page content, keeping only the one in the header.

Reply:
Fixed with an upstream patch. See https://github.com/ec-europa/rdf_entity/pull/125